### PR TITLE
keystrokes v5.0.1, options to disable signs, banners, & beacon beams

### PR DESCRIPTION
**Pull request by asbyth**
commit also reverts some changes sk1er made to chromahud that started a crashing issue with CCounter, so now chromahud no longer crashes (or atleast shouldnt)

also cleaned up a lot of classes, replaced most System.out.println(); calls with Hyperium.LOGGER calls (avoided spotify class because i dont want conor to have to deal w conflicts), and removed a bit of useless stuff again

(also realized that i closed that commit for no reason, sorry if im spamming literally anyones email)

### CODE REVIEW

File: src/main/java/cc/hyperium/mods/chromahud/ElementRenderer.java [hunk 1]
```diff
@@ -96,54 +102,40 @@ public static void draw(int x, double y, List<String> list) {
...
-                else
                     fontRendererObj.drawString(string, (int) (x / getCurrentScale() - shift), (int) (ty / getCurrentScale()), getColor(color, x), current.isShadow());
             }
             ty += 10D * currentScale;
         }
     }
 
-    // Don't shift, by the time it is here it is already shifted
+
+    //Don't shift, by the time it is here it is already shifted
```
Comment (RDIL) [inline src/main/java/cc/hyperium/mods/chromahud/ElementRenderer.java]:
```suggestion
    // Don't shift, by the time it is here it is already shifted
```

File: src/main/java/cc/hyperium/mods/guiscale/command/GuiscaleCommand.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+public class GuiscaleCommand implements BaseCommand {
+    @Override
+    public String getName() {
+        return "guiscale";
+    }
+
+    @Override
+    public String getUsage() {
+        return "/guiscale <integer>";
```
Comment (RDIL) [inline src/main/java/cc/hyperium/mods/guiscale/command/GuiscaleCommand.java]:
```suggestion
        return "/guiscale <number>";
```
so people know what we talking about

File: src/main/java/com/chattriggers/ctjs/loader/UriScheme.java [hunk 3]
```diff
@@ -1,5 +1,7 @@
 package com.chattriggers.ctjs.loader;
 
+import cc.hyperium.Hyperium;
+
```
Comment (RDIL) [inline src/main/java/com/chattriggers/ctjs/loader/UriScheme.java]:
```suggestion
```

File: src/main/java/net/montoyo/mcef/utilities/Util.java [hunk 4]
```diff
@@ -312,14 +313,13 @@ public static SizedInputStream openStream(String res, String err) {
 
         try {
             long len = -1;
-            boolean failed = true;
 
             //Java 6 support
```
Comment (RDIL) [inline src/main/java/net/montoyo/mcef/utilities/Util.java]:
```suggestion
            // Java 6 support
```

**Comment by asbyth** [comment]
i'll just make a pr for keystrokes v5 and the chromahud fix i guess lol

**Pull request closed without merging**